docs/setup-android.md

```diff
@@ -22,15 +22,14 @@ In order to integrate CodePush into your Android project, please perform the fol
 1. In your `android/settings.gradle` file, make the following additions:
 
     ```gradle
-    include ':app', ':react-native-code-push'
+    include ':react-native-code-push'
```

Comment:
Without an `':app'`, the application will not work. I understand that we can do it this way
```
include ':app'
include ':react-native-code-push'. 
```
But I think it will not be obvious for most users. We only need to add `': react-native-code-push'` to the existing line.

Reply:
By default ':app' is added while creating RN projects. Hence adding the same again causes build issue.

Reply:
Yes, you are right, and therefore in the documentation, we indicate that we should make the following additions. We need to add a line with `':app'` by `,':react-native-code-push'`. I think this should show the insertion point for the user.
In other words, we don't have to add `':app'` a second time, we just have to add it.